recipes/natcap-invest/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "natcap.invest" %}
+{% set version = "3.10.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/natcap/invest/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 0d9ce7ab5a9184c1cfcb2d2ec8295216e163b61e979b6bba3246342400945bdb
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - babel
+    - cython
+    - numpy >=1.11.0,!=1.16.0,<1.22.0
```

The version will be taken from the global pinning. Only add something here if the pinning conflicts with your pinnings (currently  it doesn't). Otherwise, you won't get the correct numpy version during the build.
```suggestion
    - numpy
```